pike/test/copychunk.py

```diff
@@ -67,8 +73,9 @@ def _gen_random_test_buffer(length):
     for i in range(0, length):
         if i % length == 0 and i != 0:
             buf += '-'
+        # XXX: accidentally quadratic str concat
         buf += str(random_str_seq[random.randint(0, len(random_str_seq) - 1)])
```

Comment:
Probably out of scope, but `str(random.choice(random_str_seq))`

Reply:
yeah, lots of things could be improved about this, but i think i'll consider adding the XXX to be "done enough" for the purposes of py3 compat. I don't _really_ want to start changing the logic yet.